internal/service/report.go

```diff
@@ -325,6 +327,14 @@ func (s *ReportService) consumeReportTask(ctx context.Context, reportTask *types
 			Msg("report task verification failed, marking task as unreliable")
 	}
 
+	// reportTask.CreatedAt is in microseconds
+	var taskCreatedAt time.Time
+	if reportTask.CreatedAt != 0 {
+		taskCreatedAt = time.Unix(0, reportTask.CreatedAt*1000)
```

Comment:
Because we are using a MQ, there might be backlogs when the service is deployed so zero-value check & fallback is mandatory here.

Reply:
> Because we are using a MQ, there might be backlog messages where the `createdAt` is not specified because it is generated by an older version of the service when the new version is deployed so zero-value check & fallback is mandatory here.

I meant to replace time.Unix to time.UnixMicro 

GitHub's review tool is hard to use 😢

Reply:
fixed in fee1a31da521fd02821b509be37e72d0c975537c